airbyte-ci/connectors/pipelines/pipelines/helpers/connectors/modifed.py

```diff
@@ -24,12 +24,10 @@ def _find_modified_connectors(
     file_path: Union[str, Path], all_connectors: Set[Connector], dependency_scanning: bool = True
 ) -> Set[Connector]:
     """Find all connectors impacted by the file change."""
+    active_connectors = {conn for conn in all_connectors if conn.support_level != "archived"}
     modified_connectors = set()
 
-    for connector in all_connectors:
-        if connector.support_level == "archived":
-            main_logger.info(f"Skipping connector '{connector}' due to 'archived' support level.")
-            continue
+    for connector in active_connectors:
```

Comment:
I agree that the per connector logging message is quite annoying. Would you mind keeping a single log line like `f"Skipping {len(all_connectors) - len(active_connectors)} archived connectors"`. I'd love to keep it explicit somewhere that airbyte-ci filters out archived connectors.

Reply:
Makes sense, I added the log as you suggested! 

I noticed it was being printed multiple times because I had added the mapping of active connectors to the `_find_modified_connectors` function, which runs on a loop for each modified file. Fixed it by moving the mapping to the outer `get_modified_connectors` function, so now we check for and remove `archived` connectors once before entering the loop.